src/AutoRest.CSharp/Output/Models/Requests/MgmtLongRunningOperation.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Linq;
+using AutoRest.CSharp.Generation.Types;
+using AutoRest.CSharp.Input;
+using AutoRest.CSharp.Output.Builders;
+using AutoRest.CSharp.Output.Models.Serialization;
+using AutoRest.CSharp.Output.Models.Types;
+using Azure;
+using Azure.Core;
+
+namespace AutoRest.CSharp.Output.Models.Requests
+{
+    internal class MgmtLongRunningOperation: TypeProvider
```

Comment:
Should this go into the mgmt directory? Same for other models

Reply:
are we going to group / separate lowlevel vs dataplane? If so then it might make sense to declare what are the top level types and group all files accordingly including a common? We can take this up in a future refactoring once we see how it ends up

Reply:
I think it might make sense. You already created the Mgmt directory in you or, let's use it.